ctp2_code/gs/gameobj/bldque.cpp

```diff
@@ -643,6 +641,7 @@ bool BuildQueue::BuildFront(sint32 &shieldstore, CityData *cd, const MapPoint &p
 				DPRINTF(k_DBG_GAMESTATE, ("Deducting %i shields for empty queue in city of %lx\n", s, m_city.m_id));
 				shieldstore = s;
 			}
+			assert(shieldstore >= 0);
```

Comment:
This should be Assert with capital A. That's the one we use here in the code. It raises that assert message box and makes also sure that the assert is logged in the debug logs with the stack trace.

Reply:
Good point... assert (lowercase) is still used in a couple of other places. I will fix them all at once.